### PR TITLE
Add design assets

### PR DESCRIPTION
**Pull request by elliot-butler-ho**
Also:
- Updates 'Get started' section navigation
- Adds Mural AUP PDF

### CODE REVIEW

File: apps/docs/src/common/pages/get-started/start-prototype.tsx [hunk 1]
```diff
@@ -15,16 +15,12 @@ const Page: FC<PageProps> = ({ location }) => (
     <div className="govuk-grid-column-one-quarter">
       <NavigationMenu items={[
         {
-          href: '/get-started/start-prototype',
-          text: 'Starting a new prototype'
+          href: '/get-started/prototyping',
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/get-started/start-prototype.tsx:18]: Looks like you want to rename this file to `prototyping.tsx`.

Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/get-started/start-prototype.tsx:18]: Oh sorry, I can see you've added that file.

**Comment by daniel-ac-martin** [comment]
Preview: https://609cf21e3ff852b449d64ff3--hods-poc.netlify.app/get-started

**Comment by daniel-ac-martin** [comment]
This LGTM except that I can't see where you've linked to the Mural AUP PDF.

File: apps/docs/src/common/pages/get-started/design-assets.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const applyPrototype1 = require('../../../../assets/images/prototype-screen01.png').default;
+const applyPrototype2 = require('../../../../assets/images/prototype-screen02.png').default;
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/get-started/design-assets.tsx]: These lines should be removed.

And maybe you want import the Mural PDF here instead?

Comment (karypun) [reply apps/docs/src/common/pages/get-started/design-assets.tsx]: @daniel-ac-martin there seems to be two mural pdfs in the assets folder: Mural_AUP.pdf and mural-aup.pdf. Could  they just sit in the assets level? They don't need to be imported to be linked anywhere on the site as it exists as a holding place for them to be linked from the Mural software.

**Review by karypun — APPROVED** [review]

File: apps/docs/src/common/pages/get-started/design-assets.tsx (hunk 2 above)
Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/get-started/design-assets.tsx]: I've fixed this.

**Comment by daniel-ac-martin** [comment]
Nice work @elliot-butler!

**Pull request merged**